diff --git a/BUILD.bazel b/BUILD.bazel
--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -1 +1 @@
-    module_bazel = "MODULE.bazel",
+    module_bazel = "registry_package.bazel",
diff --git a/registry_package.bazel b/registry_package.bazel
new file mode 100644
--- /dev/null
+++ b/registry_package.bazel
@@ -0,0 +1,14 @@
+"""https://gmplib.org/"""
+
+module(
+    name = "gmp",
+    version = "6.3.0",
+    bazel_compatibility = [">=7.2.1"],
+    compatibility_level = 1,
+)
+
+bazel_dep(name = "rules_cc", version = "0.2.17")
+bazel_dep(name = "bazel_skylib", version = "1.8.2")
+bazel_dep(name = "platforms", version = "1.0.0")
+bazel_dep(name = "rules_cc_autoconf", version = "0.10.1")
+bazel_dep(name = "m4", version = "1.4.21.bcr.1")
diff --git a/BUILD.bazel b/BUILD.bazel
--- a/BUILD.bazel
+++ b/BUILD.bazel
@@ -831,2 +831,3 @@
     name = "gmp",
+    alwayslink = True,
     srcs = [
